docs/guides/agents.md

```diff
@@ -0,0 +1,48 @@
+# Agents
+
+An Agent represents an actor holding identity and capable of interacting with
```

Comment:
"...an actor holding identity and..." reads weird to me somehow, "...represents a digital Identity capable of interaction with other digital Identities" ?

Reply:
Yeah agreed!